cc-snapshot

```diff
@@ -117,6 +118,18 @@ if [ "$(version "$openstackclient_version")" -lt "$(version "3.16.0")" ]; then
 fi
 ###### Check openstackclient version end ######
 
+###### Get provenance properties ######
+PROVENANCE_PROPERTIES=''
+if [ -f "$PROVENANCE_FILE" ]; then
+  for prop in $(cat $PROVENANCE_FILE | tr -d '\n' | tr -d [:space:] | tr -d '{' | tr -d '}' | tr ',' '\n');
```

Comment:
Can we not assume `jq` is installed? I am not sure if that is one of the base packages we include or not. This solution works fine though!

Reply:
I don't think we can assume `jq` is installed, and we have https://github.com/ChameleonCloud/cc-snapshot/blob/9092a719adf4570c2b5b993692a910b3cef2d054/cc-snapshot#L195-L211 for parsing vendor data. Actually, I checked and CC-CentOS7 has `jq` installed, but not for both Ubuntu images.